example/.eslintrc

```diff
@@ -2,7 +2,7 @@
     "extends": "eslint-config-nfl",
     "env": {
         "browser": true,
-        "mocha": true
+        "jasmine": true
```

Comment:
I think your original edit was fine to drop `mocha`. Can you just replace `context` with `describe` in those two files? `context` is just an alias for `describe`.